src/scripts.js

```diff
@@ -69,26 +79,50 @@ formPage.addEventListener("click", (event) => {
 const loadData = () => {
   fetchAll()
     .then((data) => {
-      const id = 8;
       const [travelersData, tripsData, destinationsDataObj] = data;
       destinationsArray = destinationsDataObj.destinations;
-      const travelersRepository = new TravelersRepository(
-        travelersData.travelers
-      );
-      const currentTravelerData = travelersRepository.getSingleTraveler(id);
+      travelersRepository = new TravelersRepository(travelersData.travelers);
       tripsRepository = new TripsRepository(tripsData.trips);
-      const currentTravelerTrips = tripsRepository.getTrips(id);
-      currentTraveler = new Traveler(currentTravelerData, currentTravelerTrips);
-      return { currentTraveler, destinationsArray };
-    })
-    .then(({ currentTraveler, destinationsArray }) => {
-      startApplication(currentTraveler, destinationsArray);
     })
     .catch((error) => console.log(`There has been an error! ${error}`));
 };
 
+const checkLogin = () => {
+  const login = document.querySelector(".login");
+  const userName = document.querySelector(".user-name");
+  const password = document.querySelector(".password");
+  const id = getIdNum(userName);
+  const idsArray = createIdArray();
+
+  if (idsArray.includes(id) && password.value === "travel") {
+    const currentTravelerData = travelersRepository.getSingleTraveler(id);
+    const currentTravelerTrips = tripsRepository.getTrips(id);
+    currentTraveler = new Traveler(currentTravelerData, currentTravelerTrips);
+    beginLogin();
+    startApplication(currentTraveler, destinationsArray);
+    removeHidden(formPage);
+  } else {
+    alert("Please enter a valid Username and Password ✌️");
+    login.reset();
+  }
+};
+
+const createIdArray = () => {
+  return travelersRepository.travelers.map((traveler) => traveler.id);
+};
+
+const getIdNum = (userName) => {
+  let string = userName.value.split("");
+  let length = string.length;
+  let id2 = length - 1;
+  let id1 = length - 2;
+  let first = string[id1];
+  let second = string[id2];
+  let value = parseInt(first + second);
```

Comment:
Look into the array.prototype method ‘splice’ to simplify code here, could bring this function down to 1-2 lines!